### ISSUE TITLE
Dracula set to light mode?  (qBitorrent Web v4.4.5)

### ISSUE THREAD

**Issue by ArduinoBen** [dracula/qbittorrent#24]
Hey.  I tried to set up Dracula for my qBittorrent docker container.  I am seeing the custom UI elements show up but for some reason the background is white.

Is there an option I need to change or have I done something wrong elsewhere?  Thanks for any help!

![image](https://user-images.githubusercontent.com/35127321/226956694-9bb37fe8-757e-428a-9803-5c4bd04f6bf4.png)

**Comment by jooleer** [comment]
Just to make sure, you tried disabling custom UI theme, deleting and re-importing the `dracula.qbtheme` file?

**Comment by bonny1992** [comment]
> Just to make sure, you tried disabling custom UI theme, deleting and re-importing the `dracula.qbtheme` file?

I may be wrong, but that's the webui, so no `qbtheme` file.

**Comment by jooleer** [comment]
> > Just to make sure, you tried disabling custom UI theme, deleting and re-importing the `dracula.qbtheme` file?
> 
> I may be wrong, but that's the webui, so no `qbtheme` file.

@bonny1992 yeah you're right, I should've said disable the theme, re-download and re-enable the webui theme in the settings
@ArduinoBen I don't know how you previously installed the theme, but you can also try following these steps 
(from [https://draculatheme.com/qbittorrent](https://draculatheme.com/qbittorrent)):

1. Clone the repository:
```
cd /opt
git clone https://github.com/dracula/qbittorrent.git
chmod -R 777 qbittorrent
```

2. Enable theme selection from menu: → Tools → Options → Web UI → Use alternative Web UI.
3. In 'Files locations' bar, you should type /opt/qbittorrent/webui .

**Comment by weberbox** [comment]
This still seems to be an issue? I can see the custom icons etc. but the theme is white. Followed the instructions to the letter and even deleted the theme and downloaded again with same issue. 

qBittorrent v4.2.5 Web UI (64-bit)

**Comment by Jebanisa** [comment]
Same issue here.

**Comment by oczkoisse** [comment]
Same issue on qBittorrent v4.3.9 Web UI
